kmm/read.py

```diff
@@ -1,13 +1,15 @@
 from pathlib import Path
 from pydantic import validate_arguments
 
-from kmm.camera_positions import camera_positions
 from kmm.kmm import kmm
 from kmm.kmm2 import kmm2
+from kmm.camera_positions import camera_positions
+from kmm.sync_frame_index import sync_frame_index
+from kmm.geodetic import geodetic
 
 
 @validate_arguments
-def read(path: Path, header_path: Path = None):
+def read(path: Path, header_path: Path):
```

Comment:
Should we call this function `read_sync_adjust`?

Reply:
`parse_kmm_wire_camera`? Then we could make another one for the sliper camera `parse_kmm_sliper_camera`

Reply:
That doesn't say what the function does imo and I prefer your argument idea to the function like
```
read_sync_adjust(
    "path/to/kmm(2)",
    header="path/to/hdr",
    adjust=kmm.WireCameraAdjustment,
    geodetic=True,
)
```
where you would switch "adjust" to `None` or `kmm.SliperCameraAdjustment`

Reply:
Okay sure :slightly_smiling_face: I thought though that in the future we might add more post processing besides sync and adjust and then the name wouldn't make sense anymore. Thoughts?

Reply:
`process_kmm`?